mmpose/core/evaluation/top_down_eval.py

```diff
@@ -250,6 +250,72 @@ def _taylor(heatmap, coord):
     return coord
 
 
+def post_dark(coords, batch_heatmaps, kernel=3):
+    """DARK post-pocessing. Implemented by udp. Paper ref: Huang et al. The
```

Comment:
> Implemented by udp

Are there any sufficient differences with the original dark post-processing? If they are almost equivalent, you can remove the explanations. If they have important differences, then you can rename it to `post_dark_udp`

Reply:
done